src/operationsClient.ts

```diff
@@ -345,11 +378,15 @@ export class OperationsClient {
    *     console.error(err);
    *   });
    */
-  listOperationsStream(request: {}, options: gax.CallSettings) {
+  listOperationsStream(
```

Comment:
How about `listOperationsAsync`? :)   (not that anyone really needs it, but why not? it's free :) )

Reply:
that's surely doable :) Thanks for suggestion, I will add one more interface.